zsh-osx-autoproxy.plugin.zsh

```diff
@@ -34,14 +34,14 @@ fi
 if (( $__ZSH_OSX_AUTOPROXY_FTP_PROXY_ENABLED )); then
     __ZSH_OSX_AUTOPROXY_FTP_PROXY_SERVER=${${__ZSH_OSX_AUTOPROXY_SCUTIL_PROXY#*FTPProxy : }[(f)1]}
     __ZSH_OSX_AUTOPROXY_FTP_PROXY_PORT=${${__ZSH_OSX_AUTOPROXY_SCUTIL_PROXY#*FTPPort : }[(f)1]}
-    export ftp_proxy="http://${__ZSH_OSX_AUTOPROXY_FTP_PROXY_SERVER}:${__ZSH_OSX_AUTOPROXY_FTP_PROXY_PORT}"
+    export ftp_proxy="ftp://${__ZSH_OSX_AUTOPROXY_FTP_PROXY_SERVER}:${__ZSH_OSX_AUTOPROXY_FTP_PROXY_PORT}"
     export FTP_PROXY="${ftp_proxy}"
 fi
 # all_proxy
 if (( $__ZSH_OSX_AUTOPROXY_SOCKS_PROXY_ENABLED )); then
     __ZSH_OSX_AUTOPROXY_SOCKS_PROXY_SERVER=${${__ZSH_OSX_AUTOPROXY_SCUTIL_PROXY#*SOCKSProxy : }[(f)1]}
     __ZSH_OSX_AUTOPROXY_SOCKS_PROXY_PORT=${${__ZSH_OSX_AUTOPROXY_SCUTIL_PROXY#*SOCKSPort : }[(f)1]}
-    export all_proxy="http://${__ZSH_OSX_AUTOPROXY_SOCKS_PROXY_SERVER}:${__ZSH_OSX_AUTOPROXY_SOCKS_PROXY_PORT}"
+    export all_proxy="socks5h://${__ZSH_OSX_AUTOPROXY_SOCKS_PROXY_SERVER}:${__ZSH_OSX_AUTOPROXY_SOCKS_PROXY_PORT}"
```

Comment:
It is not recommended to use socks5h here.

Reply:
From my experience, it's better to use socks5h, which indicates resolving hostname on the server.
When I use `curl www.google.com -x socks5://127.0.0.1:1080` I get polluted IP address.
But `curl www.google.com -x socks5h://127.0.0.1:1080` works just fine.